packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -25,15 +24,40 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   const filterTerminatedCards = (cards) =>
     cards.filter((card) => card.status !== CardStateType.TERMINATED)
 
+  const getEligibleAccounts = function* (cardId) {
+    try {
+      const data = yield call(api.getDCEligibleAccounts, cardId)
+      yield put(A.setEligibleAccounts(data))
+    } catch (e) {
+      console.error('Failed to get eligible accounts', errorHandler(e))
```

Comment:
then you would be able to set a failure here

Reply:
I have nowhere to show an error with this failure. However, the same response from above applies here too.